crates/router/src/core/customers.rs

```diff
@@ -238,15 +254,14 @@ impl CustomerCreateBridge for customers::CustomerRequest {
             phone_country_code: self.phone_country_code.clone(),
             metadata: self.metadata.clone(),
             connector_customer: None,
-            address_id: None,
             created_at: common_utils::date_time::now(),
             modified_at: common_utils::date_time::now(),
             default_payment_method_id: None,
             updated_by: None,
-            // default_billing_address: default_customer_billing_address,
-            // default_shipping_address: default_customer_shipping_address,
-            // merchant_reference_id,
+            default_billing_address: encrypted_customer_billing_address.map(Into::into),
+            default_shipping_address: encrypted_customer_shipping_address.map(Into::into),
             // status: Some(customer_domain::SoftDeleteStatus::Active)
+            version: common_enums::ApiVersion::V2,
```

Comment:
we can have this as a const value, based on the feature flag we can have the const value which will be either v1 or v2

Reply:
sure